mllib/src/main/scala/org/apache/spark/ml/feature/PCA.scala

```diff
@@ -105,7 +105,7 @@ object PCA extends DefaultParamsReadable[PCA] {
 @Experimental
 class PCAModel private[ml] (
     override val uid: String,
-    val pc: DenseMatrix)
+    private[feature] val pc: DenseMatrix)
```

Comment:
This seems like a pretty stable API to expose.  I'd be OK with keeping it public.  The only reason not to would be if we might change pc to be a distributed matrix someday.  But I don't think we will since it's supposed to be fairly small (low-rank).